packages/opentelemetry-core/src/platform/index.ts

```diff
@@ -18,3 +18,5 @@
 // to override this file to use `./browser/index.ts` when packaged with
 // webpack, Rollup, etc.
 export * from './node';
+// temporary - when testing opentelemetry-tracer-web or opentelemetry-basic-tracer force browser
```

Comment:
Is this the issue around the lerna compile step?

Reply:
fixed issue, comment removed